src/events/aws_sdk_v3.js

```diff
@@ -324,6 +373,7 @@ function getOperationByCommand(command) {
  */
 function AWSSDKv3Wrapper(wrappedFunction) {
     return function internalAWSSDKv3Wrapper(command) {
+        let responsePromise = wrappedFunction.apply(this, [command]);
```

Comment:
I this this may lead to us calling the function twice for unsupported services

Reply:
Nice spot, I have fixed it in the next commit